### DISCUSSION TITLE
Multiple clients publish to the same topic

### DISCUSSION THREAD

**Discussion by vanchain** [emqx/emqx#5958] [Q&A] [answered]
In the case of high concurrency, will multiple clients send the same topic at the same time cause resource competition?
or
Multiple clients send the same topic at the same time, which will cause a client to be rejected?

thanks！

**Comment by hjianbo** [comment, marked as answer]
It does not lead to competition for resources and results in send rejection.

In fact, when different clients send messages to the same topic, it is a completely unrelated process that may run on two CPU cores or even two nodes at the same time; a topic is just a string without a resource such as a queue that is available to publishers competing for its use.
